tests/integration/suite/daprd/workflow/workflow.go

```diff
@@ -15,7 +15,8 @@ package workflow
 
 import (
 	_ "github.com/dapr/dapr/tests/integration/suite/daprd/workflow/continueasnew"
-	_ "github.com/dapr/dapr/tests/integration/suite/daprd/workflow/crossapp"
+	_ "github.com/dapr/dapr/tests/integration/suite/daprd/workflow/crossapp/activity"
+	_ "github.com/dapr/dapr/tests/integration/suite/daprd/workflow/crossapp/suborchestrator"
```

Comment:
```suggestion
	_ "github.com/dapr/dapr/tests/integration/suite/daprd/workflow/crossapp"
```
I think we want this to just have `crossapp` then in the `crossapp` dir we want a new file called `crossapp.go` with the following content:
```
/*
Copyright 2025 The Dapr Authors
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
    http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/

package crossapp

import (
	_ "github.com/dapr/dapr/tests/integration/suite/daprd/workflow/crossapp/activity"
	_ "github.com/dapr/dapr/tests/integration/suite/daprd/workflow/crossapp/suborchestrator"
)
```